.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: php
+
+php:
+  - "7.2"
```

Comment:
I think it's better to be consistent with the version of php you've chosen (the 8)